src/components/MobileShell/MobileShell.module.css

```
@@ -0,0 +1,130 @@
+  /* Intentional breathing room: the height-derived board caps at 64px rows,
+     so the leftover space frames it instead of reading as a dead band. */
+  padding-block: clamp(12px, 4dvh, 48px);
+  transition: padding-bottom 280ms ease;
+}
+
+/* While the Overlay panel is open it covers the viewport up to
+   --mobile-panel-cover (dock included) — pad the stage by the part of that
+   cover that overlaps it (the dock already sits in flex flow below). */
+.shell[data-mobile-panel="overlay"] .stage {
+  padding-bottom: calc(
+    var(--mobile-panel-cover, 55dvh) - var(--mobile-dock-height, 104px) -
```

Stylelint (17.12.0)

[error] Unexpected newline after "-" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)